Makefile

```diff
@@ -33,9 +33,9 @@ libcrt0/libcrt0.a: libcrt0/crt0.o
 	llvm-ar -qc $@ libcrt0/crt0.o
 
 # Common C functions
-LIBOBJS=libcommon/lib.o libcommon/proto.o
+LIBOBJS=libcommon/assert.o libcommon/led.o libcommon/lib.o libcommon/proto.o libcommon/qemu_debug.o
 libcommon/libcommon.a: $(LIBOBJS)
-	llvm-ar -qc $@ libcommon/lib.o libcommon/proto.o
+	llvm-ar -qc $@ $(LIBOBJS)
 $(LIBOBJS): include/types.h include/tk1_mem.h include/lib.h include/proto.h
```

Comment:
Strictly this should be include the `led.h` and `assert.h`;
`$(LIBOBJS): include/types.h include/tk1_mem.h include/lib.h include/proto.h include/led.h include/assert.h`